### PR TITLE
Add time picker input 

### LINKED ISSUE #1852: Exploring better time selector
Ref - https://github.com/bigbinary/neeto-engineering-ui-ux/issues/174

@ajmaln _A Please assign this to someone

### PR DESCRIPTION
**Pull request by AkhilXavier95**
- Fixes #1852 

**Description**
Added: `TimePickerInput` component

**Checklist**

- [x] I have made corresponding changes to the documentation.
- [x] I have updated the types definition of modified exports.
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

### CODE REVIEW

File: src/components/TimePickerInput/index.jsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import React from "react";
+
+import { useId } from "@reach/auto-id";
```
Comment (ajmaln) [inline src/components/TimePickerInput/index.jsx:3]: We can use `useId` from react. Ref: https://github.com/bigbinary/neeto-ui/pull/1900/files#diff-f9a1ff39077718831e879c270383894982bfe3ce450eab316e8473dc94cf88ce

**Review by ajmaln — APPROVED** [review]
@AkhilXavier95 _a Few suggestions, rest LGTM. Please update and assign to UI team

**Comment by AkhilXavier95** [comment]
@ajmaln _a, updated the PR with the suggestion mentioned.

**Comment by praveen-murali-ind** [comment]
I have made the following changes.

- Added `size` prop and styles.
- Added `nakedInput` prop and styles.
- Added focus visible and error styles.
- Added disabled state styles.
- Refactored CSS and component markup.

---

@AkhilXavier95 @ajmaln _a Please cross check the props for this component. I think we can get rid of these props, instead we can provide a link to the `react-time-picker` doc.

Screenshot 2023-10-26 at 2 07 39 PM

Screenshot 2023-10-26 at 2 41 16 PM

**Comment by ajmaln** [comment]
@AkhilXavier95 _a Let's remove those props and add a link to the doc. Check for other props also that we can remove from. Also remove from the types

**Comment by AkhilXavier95** [comment]
@ajmaln _a removed the props. and updated the doc. Please review.

**Review by ajmaln — APPROVED** [review]
Looks good

**Pull request merged**